engines/bladerunner/font.cpp

```diff
@@ -37,6 +37,48 @@ Font::~Font() {
 	close();
 }
 
+#if SUBTITLES_SUPPORT
+#if SUBTITLES_EXTERNAL_FONT
+// for external FON font file / subtitles support
+bool Font::openFromStream(Common::ScopedPtr<Common::SeekableReadStream> &stream, int screenWidth, int screenHeight, int spacing1, int spacing2, uint16 color) {
+    reset();
+
+	_screenWidth = screenWidth;
+	_screenHeight = screenHeight;
+	_spacing1 = spacing1;
+	_spacing2 = spacing2;
+	_color = color;
+
+	if (!stream) {
+		return false;
+	}
+	_characterCount = stream->readUint32LE();
+    debug("Font's character count: %d", _characterCount);
```

Comment:
Debug leftover? And indentation is wrong.

Reply:
Removed